milestone3/frontend/src/App.js

```diff
@@ -0,0 +1,34 @@
+import './App.css';
+import Navbar from './components/navbar.jsx';
+import AboutMe from './components/aboutMe.jsx';
+import Home from './components/home.jsx';
+import RecipePage from './components/recipePage.jsx';
+import recipes from "./recipeData.js";
+import { BrowserRouter as Router, Routes, Route,} from "react-router-dom";
+
+function App() {
+  return (
+    <Router>
+        <Navbar />
+        <div className="spacing">         
+          <Routes>
+            <Route exact path="/" element={<Home />} />
+            <Route path="/aboutMe" element={<AboutMe />} />
+            {recipes.map(recipe =>
+              <Route exact path={recipe.link} element={
+                <RecipePage 
+                  name={recipe.name} 
+                  desc={recipe.info}
+                  image={recipe.image}
+                  ingredients={recipe.ingredients}
+                  steps={recipe.instructions}
+                />
+              }/>
+            )};
```

Comment:
Nice! Not many people were able to figure out this map and it will certainly come in handy when you connect to a db.